bcipy/helpers/triggers.py

```diff
@@ -694,17 +229,20 @@ class TriggerHandler:
     def __init__(self,
                  path: str,
                  file_name: str,
-                 flush_sens: FlushFrequency):
+                 flush: FlushFrequency):
         self.path = path
         self.file_name = f'{file_name}.txt'
-        self.flush_sens = flush_sens
+        self.flush = flush
         self.triggers = []
 
         if os.path.exists(self.file_name):
             raise Exception(f"[{self.file_name}] already exists, any writing "
                             "will overwrite data in the existing file.")
 
-        self.file = open(self.file_name, 'w+')
+        self.file = open(f'{self.path}/{self.file_name}', 'w+')
+
+    def __exit__(self):
```

Comment:
The `__exit__` method needs some additional parameters:
   `def __exit__(self, exc_type, exc_value, traceback)`